### DISCUSSION TITLE
Redisotel is there option to format span operation name?

### DISCUSSION THREAD

**Discussion by darklongnightt** [redis/go-redis#2322] [Q&A]
Currently span operation names are set as `get`, `set`, `del`, is there way to tweak it such that it is `redis.get`

**Comment by RangelReale** [comment]
looks like this is hardcoded.

**Comment by vmihailenco** [comment]
It is hardcoded, but a PR with an ability to customize names is welcome.

**Reply by RangelReale** [reply]
Created [an MR](https://github.com/redis/go-redis/pull/2444) for this.
